### PR TITLE
[wip] chobby offline mode

### PR DESCRIPTION
**Pull request by DeinFreund**
I've added caching of chobby version and engine version for offline play.

### CODE REVIEW

**Comment by Licho1** [comment]
I think it is dangerous...
For steam, you want to silently run even if nothing online works...

File: Shared/PlasmaShared/GlobalConst.cs [hunk 1]
```diff
@@ -34,6 +34,7 @@ static GlobalConst()
 #else
             Mode = ModeType.Local;
 #endif
+            Mode = ModeType.Live;
```
Comment (Licho1) [inline Shared/PlasmaShared/GlobalConst.cs]: You shall not pass! This will make test version of the game screw the live databas!

Comment (DeinFreund) [reply Shared/PlasmaShared/GlobalConst.cs]: this is just so i can test locally and why i wrote WIP so you don't merge anything

**Comment by DeinFreund** [comment]
Even though Chobby worked for me just fine offline. If it turns out it doesn't this is what I'd use to make it work. Maybe the message could be modified to "Unable to establish connection to server, some features will not work. Try to run the game anyway?". This assures that chobby doesn't silently start offline and then breaks without warning due to some missing download.

**Pull request closed without merging**